presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -85,15 +85,16 @@ public ColumnInfo(
         this.catalogName = requireNonNull(catalogName, "catalogName is null");
     }
 
-    public static void setTypeInfo(Builder builder, TypeSignature type)
+    public static void setTypeInfo(Builder builder, TypeSignature typeSignature)
```

Comment:
Renaming this variable makes the diff harder to read. Such refactorings, if necessary, should be done as a separate commit to be easier to review and understand when looking at history.

Keep the original name and name the one below `typeName`